04.ls/ls.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+files = Dir.glob('*').sort
```

Comment:
デフォルトでsortされますね
https://docs.ruby-lang.org/ja/latest/method/Dir/s/=5b=5d.html

```suggestion
files = Dir.glob('*')
```